app/helpers/application_helper.rb

```diff
@@ -216,6 +216,30 @@ def community_members_label
     SiteConfig.community_member_label.pluralize
   end
 
+  def meta_keywords_default
```

Comment:
**suggestion (non-blocking)** what if we explored moving these into their own helper that gets included in the `ApplicationHelper`? If we started that trend, then at some point in a separate PR we can follow suit and clean up this file a bit. I could almost see us doing the same for other "groups" of methods that are in this file, like the [`url` methods](https://github.com/forem/forem/blob/1cd89399ea53697e37bb2cc8aa5f00ec923fd350/app/helpers/application_helper.rb#L243-L269). That approach may even give us more flexibility be only including `MetaTagHelper` in areas instead of `ApplicationHelper`.

Reply:
@atsmith813 Because of how Rails works, this would only reduce the amount of added methods in places where we explicitly include helpers (e.g. service objects). We could however wrap them up in some little helper object:

```ruby
module MetaKeywords
  extend ActionView::Helpers::TagHelper

  def self.default
    return if SiteConfig.meta_keywords[:default].blank?
  
    tag.meta name: "keywords", content: SiteConfig.meta_keywords[:default]
  end

  ...
end

MetaKeywords.default
#=> "<meta name=\"keywords\" content=\"test\">"
MetaKeywords.default.html_safe?
#=> true
```

This completely takes them out of the global helper namespace and therefore reduces the surface arae of `ApplicationHelper`, but is still easy to use in views:

```ruby
<%= MetaKeywords.default %>
```

Reply:
Hm, that's an interesting idea, @citizen428! FWIW, @atsmith813, I started down this path based on [@citizen428's suggestion here](https://github.com/forem/forem/pull/10623#discussion_r499973586), but I see your point as to how the `ApplicationHelper` could start/has started to become a bit unwieldy. I think I'll be back in this SiteConfig code again later this cycle; maybe at that point, I can investigate how it feels to wrap these into a helper object and see if it's worth the refactor effort :)